package.json

```diff
@@ -25,19 +25,40 @@
     "url": "https://github.com/silvermine/videojs-airplay/issues"
   },
   "homepage": "https://github.com/silvermine/videojs-airplay#readme",
+  "browserify": {
+    "transform": [
+      "browserify-shim"
+    ]
+  },
+  "browserify-shim": {
+    "video.js": "global:videojs"
+  },
+  "dependencies": {
+    "browserify-shim": "3.8.14"
```

Comment:
Should this be a `devDependency`?

https://github.com/thlorenz/browserify-shim#you-will-always